stablehlo/tests/interpret_constant.mlir

```diff
@@ -0,0 +1,95 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: constant_op_test_i32
+func.func @constant_op_test_i32() -> tensor<2x3xi32> {
+  %0 = stablehlo.constant dense<[[1, 2, 3], [4, 5, 6]]>: tensor<2x3xi32>
+  func.return %0 : tensor<2x3xi32>
+  // CHECK-NEXT: tensor<2x3xi32>
+  // CHECK-NEXT: 1 : i32
+  // CHECK-NEXT: 2 : i32
+  // CHECK-NEXT: 3 : i32
+  // CHECK-NEXT: 4 : i32
+  // CHECK-NEXT: 5 : i32
+  // CHECK-NEXT: 6 : i32
+}
+
+// -----
+
+// CHECK-LABEL: Evaluated results of function: constant_op_test_ui32
+func.func @constant_op_test_ui32() -> tensor<2x3xui32> {
+  %0 = stablehlo.constant dense<[[1, 2, 3], [4, 5, 6]]>: tensor<2x3xui32>
+  func.return %0 : tensor<2x3xui32>
+  // CHECK-NEXT: tensor<2x3xui32>
+  // CHECK-NEXT: 1 : ui32
+  // CHECK-NEXT: 2 : ui32
+  // CHECK-NEXT: 3 : ui32
+  // CHECK-NEXT: 4 : ui32
+  // CHECK-NEXT: 5 : ui32
+  // CHECK-NEXT: 6 : ui32
+}
+
+// -----
+
+// CHECK-LABEL: Evaluated results of function: constant_op_test_bf16
+func.func @constant_op_test_bf16() -> tensor<bf16> {
```

Comment:
nit: Should we use the same values from interpret_add.mlir for each type? That way the only change will be in the operator (add vs constant) and the CHECK results. Just to be consistent.  Here and below.

Also, how about including the tests for all the supported integer types with values borrowed from add tests?

Reply:
Since ops can be unary or binary, I will borrow the special values from the lhs, rhs, and the result from `interpret_add.mlir`